upload/src/addons/SV/WarningImprovements/XF/Service/User/WarningPointsChange.php

```diff
@@ -248,7 +315,8 @@ public function  warningActionNotifications()
                     'warning_title' => $this->warning->title,
                     'warning_points' => $this->warning->points,
                     'warning_category' => $this->warning->Definition->Category,
```

Comment:
This needs to be the title, passing an object into a pharse doesn't work.

Same for 'warning' (just remove this one).